string input_image_path
---
string output_image_path
